apps/settings/js/panels/languages/languages.js

```diff
@@ -37,7 +37,6 @@ define(function(require) {
           year: 'numeric'
         });
         var timeString = d.toLocaleString(navigator.languages, {
-          hour12: navigator.mozHour12,
```

Comment:
This might be a question to UX, but should we show the default format here, or the one the user has selected?